docs/getting_started/why_use_GTFS.md

```diff
@@ -1,42 +1,104 @@
-# Why Use GTFS?
+# Reach more riders and improve public transit with GTFS
+GTFS, the open-source standard for transit data, empowers you to make public transportation more accessible and efficient.
```

Comment:
I would remove _"the open-source standard for transit data"_ since we're explaining this in the previous page